install/0000_30_machine-api-operator_15_ironic-exporter-prometheusrules.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: metal3-baremetalhost-controller
+  namespace: openshift-machine-api
+spec:
+  groups:
+  - name: metal3-baremetalhost-controller
+    rules:
+    - alert: HighCPUTemperature
+      annotations:
+        summary: "The baremetal node {{ $labels.node_name }} CPU {{ $labels.entity_id }} is too high"
+        description: "The baremetal node {{ $labels.node_name }} CPU {{ $labels.entity_id }} is too high in the past minute. Last measurement {{ $value }}"
```

Comment:
It says `in the past minute` while the for condition is `5m`.

Reply:
Changed